demo/ez-field-edit.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+        <title>ez-field-edit demo</title>
+
+        <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+        <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
+        <link rel="import" href="../ez-field-edit.html">
+        <style is="custom-style" include="demo-pages-shared-styles">
+            ez-field-edit input:invalid {
+                border: 1px solid #d92d42;
+                background-color: #fceaec;
+                box-shadow: inset 0 0 8px rgba(217, 45, 66, .25);
+            }
+
+            .ez-invalid label {
+                color: #d92d42;
+            }
+        </style>
+    </head>
+    <body>
+        <div class="vertical-section-container centered">
+            <h3>ez-field-edit demo</h3>
+            <demo-snippet>
+                <template>
+                    <form>
+                        <ez-field-edit>
+                            Simple input:
```

Comment:
put this string and `Multiple inputs` in a `h1` or something like that, this would help using the demo